src/integrators/controllers.jl

```diff
@@ -65,7 +65,7 @@ end
         q = inv(qmax)
     else
         expo = 1 / (get_current_adaptive_order(alg, integrator.cache) + 1)
-        qtmp = DiffEqBase.fastpow(EEst, expo) / gamma
+        qtmp = ^(EEst, expo) / gamma
```

Comment:
shouldn't these have a `@fastmath` (and Float32)?

Reply:
Yes, I was just seeing what's required in order to make Enzyme work on this.